tests/unit/responses/elsevier/j.nima.2019.162728_expected.yml

```diff
@@ -0,0 +1,261 @@
+copyright_holder: Elsevier B.V.
+copyright_statement: © 2019 Elsevier B.V. All rights reserved.
+copyright_year: 2019
+document_type: conference paper
+license_url:
+license_statement:
+article_type: full-length article
+journal_title: Nuclear Inst. and Methods in Physics Research A
+material: publication
+publisher: Elsevier B.V.
+year: 2020
+authors:
+- full_name: Guber, F.
+  raw_affiliations:
+  - value: Institute for Nuclear Research RAS, Moscow, Russia
+    source: Elsevier B.V.
+  emails:
+  - guber@inr.ru
+- full_name: Finogeev, D.
+  raw_affiliations:
+  - value: Institute for Nuclear Research RAS, Moscow, Russia
+    source: Elsevier B.V.
+- full_name: Golubeva, M.
+  raw_affiliations:
+  - value: Institute for Nuclear Research RAS, Moscow, Russia
+    source: Elsevier B.V.
+- full_name: Ivashkin, A.
+  raw_affiliations:
+  - value: Institute for Nuclear Research RAS, Moscow, Russia
+    source: Elsevier B.V.
+- full_name: Izvestnyy, A.
+  raw_affiliations:
+  - value: Institute for Nuclear Research RAS, Moscow, Russia
+    source: Elsevier B.V.
+- full_name: Karpushkin, N.
+  raw_affiliations:
+  - value: Institute for Nuclear Research RAS, Moscow, Russia
+    source: Elsevier B.V.
+  - value: Moscow Institute of Physics and Technology, Dolgoprudny, Moscow Region,
+      Russia
+    source: Elsevier B.V.
+- full_name: Morozov, S.
+  raw_affiliations:
+  - value: Institute for Nuclear Research RAS, Moscow, Russia
+    source: Elsevier B.V.
+  - value: National Research Nuclear University MEPhI, Moscow, Russia
+    source: Elsevier B.V.
+- full_name: Kugler, A.
+  raw_affiliations:
+  - value: Nuclear Physics Institute of CAS, R̆ez̆, Czech Republic
+    source: Elsevier B.V.
+- full_name: Mikhaylov, V.
+  raw_affiliations:
+  - value: Nuclear Physics Institute of CAS, R̆ez̆, Czech Republic
+    source: Elsevier B.V.
+  - value: Czech Technical University, Prague, Czech Republic
+    source: Elsevier B.V.
+- full_name: Senger, A.
+  raw_affiliations:
+  - value: GSI Helmholtz Centre for Heavy Ion Research, Darmstadt, Germany
+    source: Elsevier B.V.
+artid: '162728'
+title: Transverse and longitudinal segmented forward hadron calorimeters with SiPMs light readout for future fixed target heavy ion experiments
+number_of_pages:
+dois:
+- material: publication
+  doi: 10.1016/j.nima.2019.162728
+journal_volume: '958'
+journal_issue: '3'
+is_conference_paper: true
+publication_date: '2020-04-01'
+collaborations: ['NA61/SHINE, CBM and BM@N collaborations']
+documents:
+- key: j.nima.2019.162728.xml
+  url: http://example.org/j.nima.2019.162728.xml
```

Comment:
where is this coming from?

Reply:
That is from test which tests the method
```    
def attach_fulltext_document(self, file_name, url):
        self.builder.add_document(file_name, url, fulltext=True, hidden=True)
```